frontend/.eslintrc.js

```diff
@@ -8,6 +8,7 @@ const sharedRules = {
   "@typescript-eslint/no-explicit-any": "off",
   "react-hooks/exhaustive-deps": "off", // TODO: remove this line
   "react-hooks/rules-of-hooks": "off", // TODO: remove this line
+  "@typescript-eslint/explicit-module-boundary-types": "off",
```

Comment:
This rule doesn't seem helpful for us and is annoying to comply with. Lmk if anyone wants more justification for disabling it.